spec/factories/trainees.rb

```diff
@@ -209,6 +209,10 @@
       study_mode { COURSE_STUDY_MODES[:full_time] }
     end
 
+    trait :opt_in_undergrad do
+      training_route { TRAINING_ROUTE_ENUMS[:opt_in_undergrad] }
+    end
+
```

Comment:
Is that really needed?

See https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#enum-traits

```ruby
FactoryBot.build(:trainee, :opt_in_undergrad)
```

Reply:
fixed, thanks